gsi_wdl_tools/workflow_info.py

```diff
@@ -67,9 +68,28 @@ def outputs(self):
                     description = output_descriptions.get(output_file_name)
             else:
                 raise Exception('Unsupported input type')
+            
+            vidarr_label = []            
+
+            # Check if 'vidarr_label' exists in the output description
+            if 'vidarr_label' in output_descriptions[output.name]:
```

Comment:
If an output doesn't have a corresponding entry in output_meta, the script crashes on this line with an unclear (to users) message. If a corresponding output_meta entry is a requirement, check earlier and write out a clearer message, or account for this possibility